CollAction/Controllers/AdminController.cs

```diff
@@ -232,7 +232,7 @@ public IActionResult Index()
                         "<br>" +
                         "The CollAction Team has reviewed your project proposal and is very happy to share that your project has been approved and now live on www.collaction.org!<br>" +
                         "<br>" +
-                        "So feel very welcome to start promoting it! If you have any further questions, feel free to contact the CollAction Team at collactionteam@gmail.com. And don’t forget to tag CollAction in your messages on social media so we can help you spread the word(FB: @collaction.org, Twitter: @collaction_org)!<br>" +
+                        "So feel very welcome to start promoting it! If you have any further questions, feel free to contact the CollAction Team at collactionteam@gmail.com. And don’t forget to tag CollAction in your messages on social media so we can help you spread the word ( FB: <a href='https://www.facebook.com/collaction.org/'>@collaction.org</a>, Twitter: @collaction_org)!<br>" +
```

Comment:
I think you added one space too much (looking at CA-318 on JIRA).

There's a space added between FB and (: "( FB" that probably shouldn't be there.

Reply:
This is fixed in 8458770